controller_manager/test/test_chainable_controller/test_chainable_controller.cpp

```diff
@@ -101,21 +101,23 @@ controller_interface::return_type TestChainableController::update_and_write_comm
 
   for (size_t i = 0; i < command_interfaces_.size(); ++i)
   {
-    command_interfaces_[i].set_value(reference_interfaces_[i] - state_interfaces_[i].get_value());
+    (void)command_interfaces_[i].set_value(
+      reference_interfaces_[i] - state_interfaces_[i].get_value<double>().value());
```

Comment:
```suggestion
    command_interfaces_[i].set_value(
      reference_interfaces_[i] - state_interfaces_[i].get_value<double>().value());
```
do we really need that `(void)`?

Reply:
I think the (void) is to prevent the `warning: ignoring return value of ‘bool`, a QoL fix.

That said, not checking the return of `set_value` has some downside. In terms of guidelines, when is this reasonable/acceptable ?

Reply:
Sorry I missed this comment. Yes you are right @Juliaj

Regarding the return value bool, it is always true for a synchronous run of the whole system. When the components and controllers run asynchronously. This will give you the idea of whether it is successful or not.

I've added a test in https://github.com/ros-controls/ros2_control/pull/2047 testing handles between different threads.

Reply:
thank you!